todo-list-ui/src/app/Todo/todo.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-todo-component',
+  templateUrl: './todo.component.html',
+  standalone: true,
+})
+export class Todo {
+  @Input() id!: string;
```

Comment:
Let's add the `{ requried: true }` option to required Inputs: https://angular.dev/api/core/Input#required
This helps with template validation at build time